src/main/java/io/blamer/hub/controller/TokenController.java

```diff
@@ -41,8 +45,23 @@ public TokenController(final Tokens tkns) {
    * @throws Exception if something went wrong
    */
   @PostMapping
-  Mono<Void> registerToken(@RequestBody final RequestToken request)
-    throws Exception {
-    return this.tokens.add(new TokenToAdd(request).value());
+  Mono<Void> registerToken(@RequestBody final RequestToken request) throws Exception {
+    String token = request.getValue();
+    String secretKey = "your-secret-key";
+
```

Comment:
@iColdLight empty lines mean code smell, read [this](https://www.yegor256.com/2014/11/03/empty-line-code-smell.html)